packages/components/src/internal/url/URLResolver.ts

```diff
@@ -123,7 +123,11 @@ export function parsePathName(path: string) {
     let controller = null;
 
     const dash = action.indexOf('-');
-    if (dash > 0) {
+    if (action.toLowerCase().indexOf('pipeline-status-details') === 0) {
```

Comment:
Can we use `dash = action.lastIndexOf('-')` instead of this special casing?  Or do some of the actions have dashes in them in other cases?

Reply:
Confirmed that the server side logic also uses lastIndexOf, so should be safe